PayloadPkg/FirmwareUpdate/GetCapsuleImage.c

```diff
@@ -427,41 +428,48 @@ GetCapsuleImage (
     DEBUG((DEBUG_ERROR, " CapsuleInfo not found \n"));
     return EFI_NOT_FOUND;
   }
+  DevInstanace_configured = CapsuleInfo->DevInstance;
+  //DevInstance is enumerated upto configured DevInstance in "CfgData_CapsuleInformation.yaml"
+  for (CapsuleInfo->DevInstance = 0; CapsuleInfo->DevInstance <= DevInstanace_configured; CapsuleInfo->DevInstance ++)
```

Comment:
From this pacth, it would firstly to try instance 0 to get capsule image even device instance 1 is configured for capsule image, is it expected?

Reply:
Yes, That's the expected behavior.

Reply:
For example, user configured to read capsule image from NVMe 1, but it read a capsule from NVMe 0 if both NVMe have capsule image on it. I think we should try to read capsule image from the specified/configured device first.